search/engine.go

```diff
@@ -800,7 +812,7 @@ func (e *ESEngine) DoSearch(ctx context.Context, query Query, sortBy string, fro
 				if hit.Score != nil {
 					if _, hasId := filtered.HitIdsMap[hit.Id]; hasId {
 						log.Infof("Same hit found for both regular and grammar filtered results: %v", hit.Id)
-						if hit.Score != nil {
+						if hit.Score != nil && *hit.Score > 5 {
```

Comment:
Comment + Examples.

Reply:
Comment added. Unfortunately cannot remember the example.